indra/llui/llscrolllistctrl.cpp

```diff
@@ -1741,6 +1712,16 @@ void LLScrollListCtrl::setEnabled(bool enabled)
     mCanSelect = enabled;
     setTabStop(enabled);
     mScrollbar->setTabStop(!enabled && mScrollbar->getPageSize() < mScrollbar->getDocSize());
+
+    // when the table is disabled also disable its items
+    for (LLScrollListItem* item : mItemList)
+    {
+        item->setEnabled(enabled);
+        if (!enabled)
+        {
+            item->setSelected(false);
+        }
+    }
```

Comment:
This is new behavior across all tables in the UI: when the table itself is disabled/enabled then so are the items in the table.